components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/SSOConsentServiceImpl.java

```diff
@@ -624,7 +626,7 @@ private Purpose addDefaultPurpose() throws SSOConsentServiceException {
             purpose = getConsentManager().addPurpose(defaultPurpose);
         } catch (ConsentManagementException e) {
             throw new SSOConsentServiceException("Consent purpose error",
-                                                 "Error while adding purpose: " + DEFAULT_PURPOSE, e);
+                    "Error while adding purpose: " + DEFAULT_PURPOSE, e);
```

Comment:
`"Error while adding purpose: " + DEFAULT_PURPOSE,` can be moved to the previous line

Reply:
Fixed in 06271985d0d22192422630521503f04db25e5eff.